util.go

```diff
@@ -50,17 +51,21 @@ func ExtractTrailingFields(
 }
 
 func RenderMessage(args ...interface{}) string {
-	message := ""
-	for _, a := range args {
-		if message != "" {
-			message += " "
+	var message bytes.Buffer
+
+	for i, a := range args {
+		if i > 0 {
+			// message.WriteString(" ")
```

Comment:
I think we can remove this, `message.writeByte(' ')` clearly does this job.

Reply:
Removed

Reply:
Done